photon-api/src/main/scala/com/linkedin/photon/ml/optimization/game/RandomEffectOptimizationProblem.scala

```diff
@@ -135,7 +134,8 @@ object RandomEffectOptimizationProblem {
    * @return A new [[RandomEffectOptimizationProblem]] object
    */
   def apply[RandomEffectObjective <: SingleNodeObjectiveFunction](
-      linearSubspaceProjectorsRDD: RDD[(REId, LinearSubspaceProjector)],
+      data: DataFrame,
+      rEType: REType,
```

Comment:
Class is unused now, so it can be deleted completely

Reply:
We'd rather keep it since it is very likely for us to reuse it in the future, e.g. for ADs.